integration-tests/gradle/src/test/java/io/quarkus/gradle/HttpUtils.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.gradle;
+
+import io.quarkus.test.devmode.util.DevModeTestUtils;
+import io.quarkus.utilities.JavaBinFinder;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Future;
+import java.util.function.Supplier;
+
+public class HttpUtils {
+
+    private static Future<?> quarkusDev;
+
+    protected static Process doLaunch(Path jar, File output) throws IOException {
+        List<String> commands = new ArrayList<>();
+        commands.add(JavaBinFinder.findBin());
+        commands.add("-jar");
+        commands.add(jar.toString());
+        ProcessBuilder processBuilder = new ProcessBuilder(commands.toArray(new String[0]));
+        processBuilder.redirectOutput(output);
+        processBuilder.redirectError(output);
+        return processBuilder.start();
+    }
+
+    protected static String getHttpResponse(String path) {
+        return DevModeTestUtils.getHttpResponse(path, getQuarkusDevBrokenReason());
```

Comment:
We don't want `DevModeTestUtils` to have an `getHttpResponse` method. We want your new test and the existing tests that use `DevModeTestUtils.getHttpResponse` to both use `HttpUtils.getHttpResponse`.

Reply:
```DevModeTestUtils``` is a read-only file, should I modify it and delete the ```getHttpResponse``` method from it and shift all the types of ```getHttpResponse``` to ```HttpUtils``` ?

Reply:
Exactly

Reply:
All the methods in `DevModeTestUtils` that deal with http response should be moved to your new HttpUtils class

Reply:
Okkay, on it

Reply:
Last thing, let's just drop this class since it's not doing anything after all... 

The idea was initially to move things out of `DevModeTestUtils` (since it contains things that are more than dev mode), but let's leave it for later...

Reply:
This class ```HttpUtils``` is using the same ```getHttpResponse``` as that of ````QuarkusDevGradleTestBase``` which then uses ```getHttpResponse``` of ```DevModeTestUtils```, so if I'll remove it, my test will use ```getHttpResponse``` straightaway from ```DevModeTestUtils```

Reply:
```getHttpResponse``` from ```HttpUtils``` currently adds another argument of ```getQuarkusDevBrokenReason```

Reply:
That doesn't really matter. You can pass the argument from the test

Reply:
True, I'll remove it